services/chord_service/chord_service.proto

```diff
@@ -4,8 +4,9 @@ import "google/protobuf/empty.proto";
 package chord;
 
 service Chord {
-    rpc FindSuccessor (ID) returns (Nodes);
+    rpc FindSuccessor (ID) returns (Node);
```

Comment:
This function should return the first successor?

Reply:
Yes